blog/2021-08-16-state_class_total.md

```diff
@@ -4,67 +4,32 @@ authorURL: https://github.com/emontnemery
 title: "New sensor state classes: total and total_increasing"
 ---
 
-Two new state classes, `STATE_CLASS_TOTAL` and `STATE_CLASS_TOTAL_INCREASING` have been
-added. In addition, it is no longer mandatory to set the `last_reset` attribute for
-`sum` statistics to be generated. The driver for the changes is to make it easier to
-integrate with devices, like utility meters.
+A new state class, `STATE_CLASS_TOTAL_INCREASING` has been added. In addition, the
+`last_reset` attribute is removed from `SensorEntity`. The driver for the changes is to
+make it easier to integrate with devices, like utility meters.
 
 ### State classes
 
-There are now 3 defined state classes:
+There are 2 defined state classes:
 
 - `STATE_CLASS_MEASUREMENT`, the state represents a measurement in present time, for 
```

Comment:
Right now it's the name of the constant that holds the value. Should this be the value itself?

Reply:
Fixed